README.md

```diff
@@ -178,19 +191,34 @@ Use the available commands for more info on the functionality available.
 
 #### Credentials error
 
-`error creating group commands for env: develop: error fetching ec2: {"develop" "development"}: NoCredentialProviders: no valid providers in chain. Deprecated.`
+1. If sandbox/prod/staging are not in the dp cli output try unsetting `AWS_REGION` and `AWS_DEFAULT_REGION`
 
-[Ensure you have AWS credentials set up](https://github.com/ONSdigital/dp/blob/main/guides/AWS_CREDENTIALS.md).
+1. `SSOProviderInvalidToken: the SSO session has expired or is invalid`
 
-If you do not want to set up separate profiles, another option is to not specify any profiles in your `~/.dp-cli-config.yml`. That way the default credentials will be used.
+    If you see the above error, you need to re-authenticate with sign in information
 
-```yaml
-environments:
-  - name: production
-    profile:
-  - name: develop
-    profile:
-```
+1. `error fetching ec2: {Name:sandbox Profile:dp-sandbox User:ubuntu Tag:awsb CI:false ExtraPorts:{Bastion:[] Publishing:[] Web:[]}}: MissingRegion: could not find region configuration`
+
+    check that you have the correct AWS profile names in your `~/.aws/config` file (dp-sandbox, dp-staging, dp-prod, dp-ci). A sample config for `~/.aws/config` is included at the end of this guide as a reference.
+
+1. `Error: no security groups matching environment: "sandbox" with name "sandbox - bastion"`
+
+    check  ~/.aws/credentials and remove any profile information added for dp-sandbox/dp-staging/dp-prod as this is not needed
```

Comment:
This line should use commas to separate things and look like:
check ~/.aws/credentials and remove any profile information added for dp-sandbox, dp-staging and dp-prod as this is not needed